### ISSUE TITLE
Update the SDK to v9.0.1

### ISSUE THREAD

**Issue by pigedy100** [OpenFTC/OpenRC-Turbo#25]
This year's SDK was released and recently updated, it may be a good time to update this repo

**Comment by Windwoes** [comment]
Yeah, I know, it's on my list to get to. I didn't really think there was much interest in this still so it hasn't been high on the priority list.
